src/components/PackageLanguages/PackageLanguages.tsx

```diff
@@ -0,0 +1,85 @@
+import type { FunctionComponent } from "react";
+import { Link } from "react-router-dom";
+import {
+  Language,
+  LANGUAGES,
+  LANGUAGE_RENDER_MAP,
+  TEMP_SUPPORTED_LANGUAGES,
+} from "../../constants/languages";
+import { getPackagePath } from "../../util/url";
+import { LanguageSupportTooltip } from "../LanguageSupportTooltip";
+
+const sizes = {
+  sm: 5,
+  md: 6,
+  lg: 8,
+};
+
+export interface PackageLanguagesProps {
+  isRounded?: boolean;
+  languages: Partial<Record<Language, unknown>>;
+  name: string;
+  size?: "sm" | "md" | "lg";
+  version: string;
+}
+
+export const PackageLanguages: FunctionComponent<PackageLanguagesProps> = ({
```

Comment:
Would like to eventually replace the component on the package header with this one - just needs some additional styles to support that use case